Makefile

```diff
@@ -77,11 +77,11 @@ test-python-universal:
 	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration --universal sdk/python/tests
 
 test-python-go-server:
-	go build -o ${ROOT_DIR}/sdk/python/feast/binaries/goserver github.com/feast-dev/feast/go/cmd/goserver
+	go build -o ${ROOT_DIR}/sdk/python/feast/binaries/goserver github.com/feast-dev/feast/go/cmd/server
```

Comment:
should we change the name of the go binary to `server` instead of `goserver`?

Reply:
yeah that's fair